lib/rubocop/cop/lint/empty_conditional_body.rb

```diff
@@ -107,13 +107,41 @@ def correct_other_branches(corrector, node)
           end
         end
 
+        def correct_other_branches?(node)
+          return false unless node.if_type? && node.else_branch
+
+          !(!empty_if_branch?(node) && node.elsif?) && !empty_else_branch?(node)
+        end
+
+        def empty_if_branch?(node)
+          return false unless node.parent&.if_type?
+          return true unless node.parent.if_branch
+
+          node.parent.if_branch.if_type? && !node.parent.if_branch.body
```

Comment:
Can you reduce repeated method calls?

```suggestion
          return false unless (parent = node.parent)
          return true unless parent.if_type?
          return true unless (if_branch = parent.if_branch)

          parent.if_branch.if_type? && !parent.if_branch.body
```